client/state/ui/notices/actions.js

```diff
@@ -0,0 +1,49 @@
+/**
+ * Internal dependencies
+ */
+import {
+	NEW_NOTICE,
+	REMOVE_NOTICE
+} from 'state/action-types';
+
+import { uniqueId } from 'lodash';
+
+export function createNoticeAction( status, text, options = {} ) {
+	return {
+		noticeId: uniqueId(),
+		duration: options.duration,
+		showDismiss: ( typeof options.showDismiss === 'boolean' ? options.showDismiss : true ),
+		type: NEW_NOTICE,
+		status: status,
+		text: text
+	};
+}
+
+export function removeNoticeAction( noticeId ) {
+	return {
+		noticeId: noticeId,
+		type: REMOVE_NOTICE
+	};
+}
+
+export function noticesMapDispatchToProps( dispatch ) {
```

Comment:
Hmm, I guess this goes back to the desire to have easy-to-use interactions for notices across the entire application? In that case, the potential for re-use is more legitimate. I still don't think it's appropriate for `actions`, or even to be defined in the `state` directory at all. Maybe this could be included as a file in `client/lib/mixins` ?